### PR TITLE
Added docs for Airbyte Cloud and link to the CLI README

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Added the following files:

- Getting Started with Airbyte Cloud
- Core concepts
- Managing Airbyte Cloud

Updated the sidebar nav with links to the new docs and the CLI README

### CODE REVIEW

**Review by erica-airbyte — APPROVED** [review]

File: docs/cloud/getting-started-with-airbyte-cloud.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+This page guides you through setting up your Airbyte Cloud account, setting up a source, destination, and connection, and verifying the sync.
+
+
+## Set up your Airbyte Cloud account
+
+To use Airbyte Cloud, set up an Airbyte Cloud account and create a new workspace:
+
+1. If you haven't already, [sign up for Airbyte Cloud](https://cloud.airbyte.io/signup?utm_campaign=22Q1_AirbyteCloudSignUpCampaign_Trial&utm_source=Docs&utm_content=SetupGuide).
+<br> Airbyte Cloud offers a 14-day free trial with $1000 worth of credits, whichever expires first. For more information, see [Pricing](https://airbyte.com/pricing).
+2. On the [Workspaces](https://cloud.airbyte.io/workspaces) page, click **Create new workspace**.
```
Comment (nataliekwong) [inline docs/cloud/getting-started-with-airbyte-cloud.md:13]: We actually made a very recent change last week in the UI that allows them to now surpass this screen (so they have less steps in the onboarding path.). This means they won't see a screen here and dont need to create a workspace, as it's created for them automatically when they verify their email. It is still worth defining what a workspace is, but we can change the creation step.

**Review by nataliekwong — APPROVED** [review]
Awesome! Just left one comment, otherwise LGTM!

File: docs/cloud/getting-started-with-airbyte-cloud.md (hunk 1 above)
Comment (Amruta-Ranade) [reply docs/cloud/getting-started-with-airbyte-cloud.md:13]: Gotcha. And the onboarding tutorial starts after this step, right? Once they click into the new workspace?

Comment (nataliekwong) [reply docs/cloud/getting-started-with-airbyte-cloud.md:13]: They won't even need to click into the new workspace. Once they click the verify email link from their email, the first page they'll see if the /onboarding page.

Comment (Amruta-Ranade) [reply docs/cloud/getting-started-with-airbyte-cloud.md]: Updated the content. LMK if it works!

Comment (nataliekwong) [reply docs/cloud/getting-started-with-airbyte-cloud.md]: Perfect! Looks great!

**Pull request merged**